src/components/messenger/search-conversations/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import './styles.scss';
+import { Channel } from '../../../store/channels';
+import { mapMembersToString } from './utils';
+
+export interface Properties {
+  className?: string;
+  placeholder?: string;
+  directMessagesList?: Channel[];
+
+  onChange: (directMessages: Channel[]) => void;
+}
+
+interface State {}
+
+export class SearchConversations extends React.Component<Properties, State> {
+  searchConverstions = (search) => {
```

Comment:
Typo, missing an `a`
```suggestion
  searchConversations = (search) => {
```